Uppsala/UppEt13.xml

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="mss" xml:lang="en" xml:id="UppEt13">
+    <teiHeader xml:base="https://betamasaheft.eu/">
+        <fileDesc>
+
+            <titleStmt>
+                <title xml:id="t1">
```

Comment:
xml:id="t1" is not necessary (I don't think it hurts, just to avoid future typing)